lms/static/js/discussions_management/views/discussions.js

```diff
@@ -114,6 +115,8 @@
                         $('.division-scheme-item.cohort').removeClass(HIDDEN_CLASS);
                         if (this.isSchemeAvailable(ENROLLMENT_TRACK)) {
                             $('.division-scheme-item').removeClass(TWO_COLUMN_CLASS).addClass(THREE_COLUMN_CLASS);
+                        } else {
```

Comment:
This is absolutely needed, as it turns out we render initially with THREE_COLUMN_CLASS when there are no available schemes. Thus enabling cohorts was showing a three-column layout, even when no enrollment track existed.